src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__start-day--sun calendar__month-length--29">
+      <div class="calendar__day calendar__day--day--1"></div>
```

Comment:
Each day cell is not showing any information. You must add the day number in the div's content.
```suggestion
      <div class="calendar__day"></div>
```

Reply:
sure, i just replaced this , modifier to `nth-child()`everywhere where it have been used. Good idea, didn't think of that for some reason.